assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -1,116 +1,34 @@
 // Required to initialize USWDS components after page load (WaterAlert ToolTips)
 import components from 'uswds/src/js/components';
 
-import {line} from 'd3-shape';
```

Comment:
this is another one that has change significantly. The sparklines and method count columns have been removed. Also the table is rendered once, after the parameters are fetched. The state of the selection does cause changes to markup.